### DISCUSSION TITLE
Reserve a Subnet which isn't a subnet

### DISCUSSION THREAD

**Discussion by positivism1** [Azure/ipam#150] [Q&A] [answered]
Hey Team
I have a CIDR within a block I want to mark as reserved, but not exactly sure how to go about it. We have a subnet used to allocate addresses to VPN clients, but no subnet attached to it, just an ip group. So it doesn't show up under a vnet. 

Looking at the block reservations and space reservation api, I can only put a size in, where as I want to put a CIDR in and lock it away so it doesn't come up when I request next subnet. 

Also - a bit new to IPAM so I may have some terminology mixed up, apologies.

**Comment by DCMattyG** [comment, marked as answer]
Hi @positivism1, welcome and thank you for posting!

The good news is a feature is already here that will cover your use case. If you refer to my answer at the bottom of #118, support for External Networks is currently in the Development branch and available for testing. This should be pushed to Production very soon. If you'd like to give it a test and let me know if this solves your problem, that would be very much appreciated.

Looking forward to your feedback, and please let us know if there's anything else we can be of assistance with!
